spec/models/user_spec.rb

```diff
@@ -156,6 +156,11 @@
         user.valid?
         expect(user.errors[:postal_code]).to include('should be 12345 or 12345-1234')
       end
+      it 'is invalid with a zip code that starts valid but contains more digits' do
+        user = build(:user, postal_code: '12345-12345')
+        user.valid?
```

Comment:
`expect(user).not_to be_valid`

Reply:
probably change the other 2 above as well

Reply:
Can do - did so in dae7f03